src/main/java/org/usfirst/frc4904/robot/subsystems/SolenoidSubsystem.java

```diff
@@ -0,0 +1,62 @@
+package org.usfirst.frc4904.robot.subsystems;
```

Comment:
I know that I wrote this, but it should be removed, and we should point to the SolenoidSubsystem branch of standard and use that instead